lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -144,6 +168,32 @@ class AwsCompileEventBridgeEvents {
               );
             }
 
+            if (RetryPolicy) {
+              if (!shouldUseCloudFormation) {
+                throw new ServerlessError(
+                  'Configuring RetryPolicy is not supported for EventBrigde integration backed by Custom Resources. Please use `provider.eventBridge.useCloudFormation` setting to use native CloudFormation support for EventBridge.',
+                  'ERROR_INVALID_RETRY_POLICY_TO_EVENT_BUS_CUSTOM_RESOURCE'
+                );
+              }
+              RetryPolicy = _.mapKeys(
+                RetryPolicy,
+                (value, key) => key[0].toLocaleUpperCase() + key.slice(1)
+              );
+            }
+
+            if (DeadLetterConfig) {
+              if (!shouldUseCloudFormation) {
+                throw new ServerlessError(
```

Comment:
I think we can drop the `ERROR` prefix as it's already clear that it's an error code